components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/FederatedAuthenticatorConfig.java

```diff
@@ -202,4 +214,20 @@ public int hashCode() {
     public boolean isValidPropertyValue(Property property) {
         return property != null && StringUtils.isNotBlank(property.getValue());
     }
+
+    /**
+     * @return
```

Comment:
Let's add the method comments properly.

Reply:
Addressed with commit ad110cdff4ffcfe5766b31f1a09bebaf633730e6